pkg/model/iam/iam_builder.go

```diff
@@ -777,6 +789,14 @@ func addKubeRouterSrcDstCheckPermissions(p *Policy) {
 	)
 }
 
+func addKindnetSrcDstCheckPermissions(p *Policy) {
+	p.unconditionalAction.Insert(
+		"ec2:DescribeInstances",
+		"ec2:ModifyNetworkInterfaceAttribute",
```

Comment:
Based on changes from https://github.com/aojea/kindnet/pull/132, only `ec2:ModifyInstanceAttribute` should be needed.
```suggestion
```

Reply:
copy pasting 😓

Reply:
😮‍💨